relayer.go

```diff
@@ -107,7 +96,11 @@ func payToTaprootScript(taprootKey *btcec.PublicKey) ([]byte, error) {
 // Relayer is a bitcoin client wrapper which provides reader and writer methods
 // to write binary blobs to the blockchain.
 type Relayer struct {
-	client *rpcclient.Client
+	client              *rpcclient.Client
+	network             *chaincfg.Params
+	revealSatAmount     btcutil.Amount
+	revealSatFee        btcutil.Amount
+	revealPrivateKeyWIF *btcutil.WIF
```

Comment:
[Proposal]
Similar, if the same key will be used for the commit tx, proposal to rename to `privateKeyWIF`